### DISCUSSION TITLE
How to delete release completely?

### DISCUSSION THREAD

**Discussion by CaravelKit** [community/community#23022] [New to GitHub]
<p>Hello,</p>
<p>I created a new release, publish it, then I found several bugs and fixed them. But I see they are not in the latest release. So, I deleted a release, fixed a couple more bugs then created a new release with the same version. And then I found that all these fixes are not in release again! Obviously it’s because github “remembered” my previous tag. </p>
<p>What should I do now? I don’t want to change a tag (release name) because it’s important to me. But I don’t know how to make github “forget” this tag, or include updates in this current release.</p>
<p>Thanks</p>

**Comment by mpboom** [comment]
<p>You can delete a release by going to the release (as if you were viewing it), then clicking the Delete Release button.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/d/d9558a93f158d47d8ff0de71b3e3fa3816b59d59.png" data-download-href="/uploads/short-url/v0CSIpggrjKgBzZzPewKIBcl0bT.png?dl=1" title="Naamloos.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/19948032/181074248-acdacb08-946b-4384-af3a-c74ff51ef323.png" alt="Naamloos.png" data-base62-sha1="v0CSIpggrjKgBzZzPewKIBcl0bT" width="690" height="384" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/d/d9558a93f158d47d8ff0de71b3e3fa3816b59d59_2_690x384.png, https://aws1.discourse-cdn.com/github/original/2X/d/d9558a93f158d47d8ff0de71b3e3fa3816b59d59.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/d/d9558a93f158d47d8ff0de71b3e3fa3816b59d59.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/d/d9558a93f158d47d8ff0de71b3e3fa3816b59d59_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Naamloos.png</span><span class="informations">1020×568 18.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by CaravelKit** [comment]
<p>Hi,</p>
<p>thanks for your response.</p>
<p>I did exactly what you described. But when I create a new release with the same name it “catches” the commits for that deleted resource, but not current ones. This is the problem.</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/caravelkit">@caravelkit</a>,</p>
<p>Thank you for being here! It is possible to delete using git from the command line. This can be done by running the following command inside the repository locally:</p>
<p><code>git push --delete origin tagname</code></p>
<p>Running the command above should delete the tag, allowing you to create it again against the correct branch.</p>
<p>I hope this helps!</p>
<p>Best,</p>
<p>Andrea</p>

**Reply by sameenfatima78** [reply]
Thank you. Faced a similar issue and this fixed it for me.

**Reply by GDogg2000** [reply]
Yes -- thank you.  Had the same issue and this worked.

**Comment by CaravelKit** [comment]
<p>Thanks!</p>
<p>I will try this approach.</p>

**Comment by Merudo** [comment]
<p>Another way is to delete the tag locally, then</p>
<pre><code>git push origin :refs/tags/tagName
</code></pre>

**Reply by PoolloverNathan** [reply]
You don't need to delete it locally, and you can just put the tag name after the colon.

**Comment by moriak** [comment]
You can delete a Tag after delete the Release.

<img width="1165" alt="image" src="https://user-images.githubusercontent.com/7157812/229212122-fdc59647-1dfa-49e1-a369-ac10521985a5.png">
